Servers/UI/OJS.Servers.Ui/ClientApp/src/pages/submissions/SubmissionsPage.tsx

```diff
@@ -1,30 +1,11 @@
-import React, { useEffect } from 'react';
+import React  from 'react';
```

Comment:
importing React is no longer necessary, we can safely remove those imports from the project